ace-setup-mq/readme.md

```diff
@@ -0,0 +1,28 @@
+# Instructions for setup-channel.sh
+## Descriptions 
+The script `setup-channel.sh` is used to create resources and set authority properly in QMGR for ACE sensor. 
+* The script must be run with a user who belongs to mqm group and run on the machine which hosts the queue manager.
+* The AUTH_USER used in the script is the user that you want to grant authority and it doesn't need to be a privileged user. 
+## Usage
+1. Download the script `setup-channel.sh`
+2. Set it executable `chmod +x setup-channel.sh`
+3. Go to the directory and execute the script
+```sh
+    ./setup-channel.sh -q <QMGR_NAME>  -d <MQ_BIN_PATH> -u <AUTH_USER>
+```
+# Instructions for remove-resources.sh
+## Descriptions
+The script `remove-resources.sh` is used to revert the authority granted with script `setup-channel.sh` and delete the objects created in QMGR for ACE sensor. 
+* The script must be run with a user who belongs to mqm group and run on the machine which hosts the queue manager.
+* The AUTH_USER used in the script is the user that you used in `setup-channel.sh`. 
+## Usage
+1. Download the script `remove-resources.sh`
+2. Set it executable `chmod +x remove-resources.sh`
+3. Go to the directory and execute the script
+```sh
+    ./remove-resources.sh -q <QMGR_NAME> -d <MQ_BIN_PATH> [-u AUTH_USER] [-l LISTENER_NAME]  [-c CHANNEL_NAME]  [-t TOPIC_NAME]
```

Comment:
**suggestion**: For revert, can we just revert everything done by `setup-channel.sh` without passing any arguments? . I assume we can store qmgr and MQ path passed to setup-channel.sh somewhere.

Reply:
Yes, we can do that. But I don't want to bundle them tightly. When we store them in somewhere, the user may need write authority to some folder, we can't guarantee that. 
Actually, if the script runs successfully, they will get the full arguments list at the end of the log like following: 
```
INFO: If you want to delete the resources you created, please make sure you have downloaded the remove-resources.sh script and run like following: 
./remove-resources.sh -q QM1 -d /opt/mqm/bin -l INSTANA.ACE.LS -c INSTANA.ACE.SVRCONN -t INSTANA.ACE.BROKER.TOPIC
```
If only partial of the objects are created, they still need to run the script `remove-resources.sh` to delete these partial objects, so the arguments for  `remove-resources.sh` are optional.  

Maybe we can consider this after we get feedback from customer, is that ok?

Reply:
yes. The command in the log should help if user wants to rervert.